website/content/installation/_index.md

```diff
@@ -187,3 +187,11 @@ described above.
 Please take the known limitations for [layer2](https://metallb.universe.tf/concepts/layer2/#limitations)
 and [bgp](https://metallb.universe.tf/concepts/bgp/#limitations) into account when performing an
 upgrade.
+
+## Setting the LoadBalancer Class
+
+MetalLB supports [LoadBalancerClass](https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-class),
+which allows multiple LoadBalancers to co-exist. In order to set the loadbalancer class MetalLB should be listening
+for, the `--lbclass=<CLASS_NAME>` parameter must be provided to both the spekaer and the controller.
+
+The helm charts support it via the `loadbalancerClass` parameter.
```

Comment:
nits:
multiple load balancer implementations/providers (like the k8s docs) ... 
spekaer -> speaker